lib/index.js

```diff
@@ -1 +1,98 @@
-module.exports = require('./binding/node_valhalla.node');
+var valhalla = require('./binding/node_valhalla.node');
+const genericPool = require("generic-pool");
+
+module.exports = function(configStr, minWorkers, maxWorkers) {
```

Comment:
Let's make the second argument `options` which defaults to an empty object, that way we can add more options in the future without growing the argument list.